src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -801,6 +801,26 @@ public void GetCommandLine()
             Assert.True(File.Exists(_pathToArbitraryBogusFile));
 
             bool successfulExit;
+            string output = RunnerUtilities.ExecMSBuild('"' + RunnerUtilities.PathToCurrentlyRunningMsBuildExe + '"', msbuildParameters, out successfulExit);
+            Assert.False(successfulExit);
+
+            Assert.Contains(RunnerUtilities.PathToCurrentlyRunningMsBuildExe + (NativeMethodsShared.IsWindows ? " /v:diag " : " -v:diag ") + _pathToArbitraryBogusFile, output, StringComparison.OrdinalIgnoreCase);
```

Comment:
I think these these are the other way around. `GetCommandLine` does not quote msbuild.exe, but`GetCommandLineQuotedExe` quotes it